colabs/Wczytanie_danych.ipynb

```diff
@@ -846,6 +894,922 @@
           "execution_count": 13
         }
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Załadowanie nowych baz\n",
+        "\n",
+        "   **W celu załadowania nowych danych należy:**\n",
+        "\n",
+        "\n",
+        "1. Pobrać token API:\n",
+        "  * Wejść w *zespół Aktorzy* na teams\n",
+        "   * Wybrać zakładkę *pliki*\n",
+        "    * Pobrać plik *kaggle.json* - API\n",
+        "2. Utworzyć plik na swoim dysku pod nazwą \"Big Data\"\n",
+        "3. Wrzucić pobrane API na swój dysk\n",
+        "4. Uruchomić kod :)\n",
+        "\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "bfRM2_2Lcdoi"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Wczytajmy bazę oskarów: \"Best Movies Watchlist\"\n"
+      ],
+      "metadata": {
+        "id": "ybdfEcrqRZ0s"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from google.colab import drive\n",
+        "drive.mount('/content/gdrive')\n",
+        "\n",
+        "#%cd /content/gdrive/MyDrive/Big Data/kaggle.json\n",
+        "import os\n",
+        "os.environ['KAGGLE_CONFIG_DIR'] = \"/content/gdrive/MyDrive/Big Data\"\n",
+        "\n",
+        "#from google.colab import drive\n",
+        "#drive.mount('/content/drive')\n",
+        "#random.seed(123456)\n",
+        "#np.set_printoptions(precision=6, suppress=True)"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "x6UtikJb8Ta-",
+        "outputId": "d350300c-3db9-4a29-ff59-b73c3b9d82ab"
+      },
+      "execution_count": 15,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Mounted at /content/gdrive\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "! pip install kaggle\n",
+        "! mkdir ~/.kaggle #Utwórz katalog o nazwie „.kaggle”\n",
+        "! cp kaggle.json ~/.kaggle/ # Skopiuj plik „kaggle.json” do tego nowego katalogu\n",
+        "! chmod 600 ~/.kaggle/kaggle.json #Przydziel wymagane uprawnienia do tego pliku."
```

Comment:
Te opisy nie mówią nic więcej ponad to co kod obok robi. Usunąć te komentarze.

Reply:
Zadanie Emilki